src/Avalonia.Controls/ColumnDefinition.cs

```diff
@@ -108,5 +109,22 @@ public GridLength Width
         internal override GridLength UserSizeValueCache => this.Width;
         internal override double UserMinSizeValueCache => this.MinWidth;
         internal override double UserMaxSizeValueCache => this.MaxWidth;
+
+        /// <inheritdoc/>
+        public bool Equals(ColumnDefinition other)
+            => Hash(this) == Hash(other);
```

Comment:
This and other methods should be in a curly brace format.

Reply:
Why exactly?

Reply:
Oops, Github didn't update my review message. It's for consistency since we use the regular curly braces on other Equals method (like these https://github.com/AvaloniaUI/Avalonia/blob/1f16de477e4317a125e85f16430457911df7f5b6/src/Windows/Avalonia.Win32.Interop/Wpf/IntSize.cs , https://github.com/AvaloniaUI/Avalonia/blob/b9edd8c17d1b2ca08d62d45eebcd6d36a95d4523/src/Avalonia.Visuals/VisualTree/TransformedBounds.cs , etc)

Reply:
It's not consistent, I have seen it in a few places. 

https://github.com/AvaloniaUI/Avalonia/blob/a8d5814a10198b0c42fff3a5b13d24eccb77703a/src/Avalonia.Visuals/Media/Brush.cs#L87

https://github.com/AvaloniaUI/Avalonia/blob/03076a46c95fb27ef48d826d3c3135054b579eec/src/Avalonia.Visuals/Media/DrawingContext.cs#L139

But I can change it if its required. I think it should be enforced in the editorconfig if that's possible ?

Reply:
@JaggerJo I agree, it needs to be enforced in editorconfig or `omnisharp.json`. Though if one inclines to make it consistent repo-wide then it'll be awesome :) I'll retract this change request.